### PR TITLE
3band indices

### PR DESCRIPTION
**Pull request by MATRIX4284**
Added Class for supporting three band differential normalized indices e.g.[Green-Blue NDVI]

### CODE REVIEW

File: torchgeo/transforms/indices.py [hunk 1]
```diff
@@ -241,3 +241,60 @@ def __init__(self, index_nir: int, index_vre1: int) -> None:
             index_vre1: index of the Red Edge band, B5 in Sentinel 2 imagery
         """
         super().__init__(index_a=index_nir, index_b=index_vre1)
+
+
+class AppendTribandNormalizedDifferenceIndex(Module):
```
Comment (calebrob6) [inline torchgeo/transforms/indices.py:246]:
```suggestion
class AppendTriBandNormalizedDifferenceIndex(Module):
```

File: torchgeo/transforms/indices.py [hunk 2]
```diff
@@ -241,3 +241,60 @@ def __init__(self, index_nir: int, index_vre1: int) -> None:
...
+    """Append normalized difference index involving 3 bands as channel to image tensor.
+
```
Comment (calebrob6) [inline torchgeo/transforms/indices.py:248]: Can you describe the format of this in the docstring, e.g. (band_a - (band_b + band_c)) / (band_a + band_b + band_c)

File: torchgeo/transforms/indices.py [hunk 3]
```diff
@@ -241,3 +241,60 @@ def __init__(self, index_nir: int, index_vre1: int) -> None:
...
+            band_b: difference band tensor component 1
+            band_c: difference band tensor component 2
+
+        Returns:
+            the index
+        """
+        return (band_a - (band_b + band_c)) / ((band_a + band_b + band_c) + _EPSILON)
+
+    def forward(self, sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
+        """Compute and append normalized difference index to image.
```
Comment (calebrob6) [inline torchgeo/transforms/indices.py:280]:
```suggestion
        """Compute and append tri-band normalized difference index to image.
```

**Review by calebrob6 — CHANGES_REQUESTED** [review]

File: torchgeo/transforms/indices.py (hunk 3 above)
Comment (MATRIX4284) [reply torchgeo/transforms/indices.py]: done

File: torchgeo/transforms/indices.py (hunk 2 above)
Comment (MATRIX4284) [reply torchgeo/transforms/indices.py:248]: done

File: torchgeo/transforms/indices.py (hunk 1 above)
Comment (MATRIX4284) [reply torchgeo/transforms/indices.py]: done

**Comment by calebrob6** [comment]
Can you add tests for this

**Comment by MATRIX4284** [comment]
> Can you add tests for this
done

**Review by adamjstewart — APPROVED** [review]

**Review by calebrob6 — APPROVED** [review]

**Pull request merged**